staging/README.md

```diff
@@ -6,15 +6,22 @@ top-level k8s.io repositories.
 
 Repositories currently staged here:
 
-- [`k8s.io/apiextensions-apiserver`](https://github.com/kubernetes/apiextensions-apiserver)
 - [`k8s.io/api`](https://github.com/kubernetes/api)
+- [`k8s.io/apiextensions-apiserver`](https://github.com/kubernetes/apiextensions-apiserver)
 - [`k8s.io/apimachinery`](https://github.com/kubernetes/apimachinery)
 - [`k8s.io/apiserver`](https://github.com/kubernetes/apiserver)
+- `k8s.io/cli-runtime`
 - [`k8s.io/client-go`](https://github.com/kubernetes/client-go)
-- [`k8s.io/kube-aggregator`](https://github.com/kubernetes/kube-aggregator)
 - [`k8s.io/code-generator`](https://github.com/kubernetes/code-generator)
+- [`k8s.io/csi-api`](https://github.com/kubernetes/csi-api)
+- [`k8s.io/kube-aggregator`](https://github.com/kubernetes/kube-aggregator)
+- `k8s.io/kube-controller-manager`
+- `k8s.io/kube-proxy`
+- `k8s.io/kube-scheduler`
+- `k8s.io/kubelet`
```

Comment:
@stantonxu Thanks for your first contribution to Kubernetes! :+1: 

@neolit123 @dims since we are already in code freeze, wdyt about waiting until the repos are created before adding them here? The request for creation already exist in https://github.com/kubernetes/org/issues/56 and https://github.com/kubernetes/org/issues/58.